packages/database/__tests__/utils/extractChangelogFromSnapshotRecords.test.ts

```diff
@@ -0,0 +1,164 @@
+import { type ChangeLog } from '../../src/models/ChangeLog';
```

Comment:
Should this test file be under `__tests__/utils/audit` to match folder structure completely?